Assets/SEE/UI/Tooltip.cs

```diff
@@ -0,0 +1,359 @@
+﻿using System;
+using System.Linq;
+using DG.Tweening;
+using Michsky.UI.ModernUIPack;
+using SEE.GO;
+using SEE.Utils;
+using TMPro;
+using UnityEngine;
+
+namespace SEE.UI
+{
+    /// <summary>
+    /// A box containing additional information which usually appears after hovering over any element for an
+    /// amount of time and disappears when moving the pointer away from the element.
+    ///
+    /// It should be used via its static methods <see cref="ActivateWith"/> and <see cref="Deactivate"/>.
+    /// You should not add this component to any game object yourself.
+    ///
+    /// Call <see cref="ActivateWith"/> when the mouse cursor enters the region of an object which should
+    /// display a tooltip, and call <see cref="Deactivate"/> when the mouse cursor leaves the region.
+    /// This class will render a tooltip on the screen with the given text if the mouse cursor stays in the region
+    /// without moving for a certain amount of time.
+    /// </summary>
+    public class Tooltip : PlatformDependentComponent
+    {
+        // TODO: Make this work on VR.
```

Comment:
Shouldn't this require a reference to an issue? I wonder why this has passed our CI checks.

Reply:
I fixed it assuming that the unity is seconds (looks like the DoTween API uses that unit).

Reply:
I think this one shouldn't be resolved yet, I have to look at the CI not working here.

Reply:
Maybe because this file existed previously and was just moved to another location?